regression/cbmc-java/NullPointerException2/Test.java

```diff
@@ -0,0 +1,18 @@
+class B extends RuntimeException {}
+
+class A {
+    int i;
+}
+
+public class Test {
+    public static void main(String args[]) {
+        A a=null;
+        try {
+	    a.i=0;
+            throw new B();
```

Comment:
Is there some importance in these unreachable throws? If so add a comment explaining what they're for